specs/sql_query_specs.ts

```diff
@@ -0,0 +1,33 @@
+import { describe, expect, it } from './lib/common';
+import SqlQuery, { TimeRange } from '../src/sql_query';
+import moment from "moment";
+import { RawTimeRangeStub } from './lib/raw_time_range_stub';
+
+describe("Query SELECT with $timeFilterByColumn and range with from and to:", () => {
+  const query = "SELECT * FROM table WHERE $timeFilterByColumn(column_name)";
+  const range: TimeRange = {
+    from: moment('2018-12-24 01:02:03'),
```

Comment:
`moment` value will vary depending on timezone, so those tests will fail if run in different tz. I suppose, it should be converted to particular tz or value in `expect` should be generated from `moment`